index.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: #f42;
+}
```

Comment:
- [ ] In order to make the styling work, you should create a `<h1>` tag in the HTML file.